documents/privacy.md

```diff
@@ -0,0 +1,301 @@
+    
+-   They allow for more accurate location matching.
+
+However, our expected accuracy of location measurements is only \~10m. Allowing for this level of inaccuracy for both parties involved in an exposure notification suggests that we should be looking to match at a radius of 20m, with a resulting match area of approx 1250 sqm, or 50 of these 9 character geohashes.
+
+With the app having to compute 50x as many hashes, we have to reduce the cost of the slow hash to compensate, and this effectively eliminates the benefits of the increased entropy.
+
+Although we have nor ruled out a move to 9 character geohashes in future (as it would give us slightly more accurate matching, and therefore reduced false positives), we have not yet established a clear justification for the increased complexity (and it may be better to invest the effort into even more secure approaches - as described below).
+
+The result of the above is that our current implementation uses 8 character geohashes.
+
+### Corner Cases
+
+Where a location is close to the corner of a geohash, and allowing for some inaccuracy in location data, there is a risk that someone else very close by, ends up reporting a different location, that maps to a different geohash.
+
+We resolve this issue in the Mobile App by using the user's exact location to determine a set of "nearby" geohashes, and checking them all for matches.
+
+<img src="..\images\NearbyGeohashes.png" alt="NearbyGeohashes" style="zoom:50%;" />
+
+The definition of a "nearby" geohash is any geohash that has a point within 10m of the user's reported location.
+
+On average, there will be 2.3 such geohashes, though the number is any given case could vary from 1 to 6 (and possibly even more in very northern or southern latitudes).
+
+This results in a match area of around 1660 sqm at the equator, 1200 sqm at 45 degrees north, so approximately the same area as a 2om radius circle, but with a slightly less predictable shape.
+
+![GeohashDistanceChart](..\images\GeohashDistanceChart.png)
+
+The chart on the right shows how this Geohash based matching results in the probability of a match / non-match for various  distances between two reported points.
+
+The orange line shows what happens when GPS data ponts are compared directly. The yellow line shows what is achieved with a geohash tile size of 8, and a radius for "nearby" geohashes of 10m.
+
+By moving to 9 character geohashes (as discussed above), we could bring the matching profile much closer to the orange line. This is something we may consider in future.
+
+### Time window size
+
+Due to battery constraints (and to a lesser extent, concerns about bandwidth and storage), we record a location data point only once every 5 minutes.
+
+There is no synchronization between the times at which two users record their location information, therefore we must consider two point to match if they fall within 5 minutes of each other.
```

Comment:
therefore we must consider two point to match should be therefore we must consider two **points** to match